gen-kube-scheduler/kube-scheduler-api-ref.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+#### WORK-IN-PROGRESS
+
+# git clone https://github.com/kubernetes/kubernetes.git
+
+# copied staging, src/k8s.io/kube-scheduler
+# /tmp/tmpdc3iihix/src/k8s.io/kubernetes/staging/src/k8s.io/kube-scheduler
+
+# Change this to installation directory of gen-crd-api-reference-docs
+GEN_DOCS=/tmp/gen-crd-api-reference-docs
+
+# Change this to where the website repository is cloned.
+K8S_WEBSITE=/website
+
+K8S_SCHEDULER=k8s.io/kube-scheduler
+
+# table style substitutions
+TABLE_SUB='<div class=\"table-responsive\"><table class=\"table table-bordered\">'
```

Comment:
would this bootstrap classes work in the generated file? I don't see any reference to the bootstrap CSS.

Reply:
Yes, possibly. I plan to close this PR for now and write up the steps I took in the issue.